src/targets/github.ts

```diff
@@ -422,6 +334,25 @@ export class GithubTarget extends BaseTarget {
     }
   }
 
+  /**
+   * Publishes the release draft.
+   *
+   * @param release_id Release object ID
+   */
+  public async publishReleaseDraft(release_id: number) {
+    if (isDryRun()) {
```

Comment:
Feels like this pattern can use something like a [decorator](https://www.typescriptlang.org/docs/handbook/decorators.html):
```js
@noopInDryRun
```